app/globals.css

```
@@ -160,6 +163,11 @@ body {
     color: var(--color-theme-pcne-fg);
 }
 
+@utility icon-theme-aws-saa {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)